INSTALL

```diff
@@ -33,6 +33,11 @@ some of the steps manually.  The more detailed description in the other
 sections of this guide will help you do that, so please refer to those
 sections if you need to.
 
+  0. Before you start: Remacs uses cargo for it's build process. if
```

Comment:
`It's` is a conjunction. You want `its`.

Reply:
👍 will tweak